src/main/python/powerFactory2json/pf2json.py

```diff
@@ -52,15 +60,27 @@ def get_attribute_dicts(raw_elements, attributes_to_include):
     Creates a list with an attribute dictionary for each raw PowerFactory element
     """
     elements = []
+    pf_edges = ["ElmLne", "ElmCoup"]
     for raw_element in raw_elements:
         element = get_attribute_dict(raw_element, attributes_to_include)
 
-        # export connected elements of nodes, transformers and lines to get grid topology
-        if (raw_element.GetClassName() in ["ElmTerm", "ElmTr2", "ElmTr3", "ElmLne", "ElmCoup"]):
+        # export connected elements of nodes and transformers
+        if (raw_element.GetClassName() in ["ElmTerm", "ElmTr2", "ElmTr3"]):
             element["conElms"] = []
             for con_elm in raw_element.GetConnectedElements():
                 element["conElms"].append(get_attribute_dict(con_elm, attributes4export["conElms"], True))
 
+        # export ids of nodes the edges are connected to
+        if (raw_element.GetClassName() in pf_edges):
```

Comment:
What is the reason, you treat lines and switches differently, here? I see, the bus information are included in `"conElms"`, but why don't you choose a similar way or at least naming here?

Reply:
Tbh I went for the conElms approach because i missed that i can get the ids via `edge.bus1` more directly. This way is much more convenient since I don't have to perform a dozen checks for whether there are exactly two connected elements and whether these are configured correctly which made the code much more elaborate and harder to understand. I then excluded the exporting of `conElms`  `lines` and `switches` since I don't need the parameter anymore but I still do need the parameter for nodes as there is no more convenient way (for checking if there is an external net at the node). I will probably scratch the parameter for transformers as welll. 

Regarding naming I sticked relatively closely to the pf variable naming as this is the way we name all other variables. Also `conElms` was the connected element as an object and with `bus[12]Id` I only export the id of the element.